pkg/cli/admin/nodeimage/create.go

```diff
@@ -71,6 +86,13 @@ var (
 
 		# Specify a custom image name
 		  oc adm node-image create --o=my-node.iso
+
+		# Create an ISO to add a single node without using the configuration file
+		  oc adm node-image create --mac-address=00:d8:e7:c7:4b:bb
+
+		# Create an ISO to add a single node without using the configuration file
```

Comment:
nit: `Create an ISO to add a single node with a root device hint and without using the configuration file`